eng/pipelines/libraries/build-job.yml

```diff
@@ -54,15 +54,12 @@ jobs:
           - ${{ format('{0}_product_build_{1}{2}_{3}_{4}', parameters.runtimeFlavor, 'linux', parameters.osSubgroup, 'x64', parameters.liveRuntimeBuildConfig) }}
 
       variables:
-        - _skipTestRestoreArg: /p:SkipTestRestore=true
-        - _buildAction: -build
+        - _addtionalBuildArguments: ''
         - ${{ parameters.variables }}
         - ${{ if eq(parameters.osGroup, 'WebAssembly') }}:
           - EMSDK_PATH: $(Build.BinariesDirectory)/emsdk
         - ${{ if eq(parameters.runTests, true) }}:
-          - _buildAction: -build -buildtests /p:ArchiveTests=true
-          - ${{ if ne(parameters.framework, 'allConfigurations') }}:
-            - _skipTestRestoreArg: /p:SkipTestRestore=false
+          - _addtionalBuildArguments: /p:IncludeLibrariesTestSubset=true /p:ArchiveTests=true
```

Comment:
I thought we preferred using actions, like `buildtests`? That is what we have been leaning towards always.

Reply:
The `buildtests` action is removed with this PR (that's why I won't merge this before 4/6 and add it to https://github.com/dotnet/runtime/issues/33821). The `buildtests` action was never a correct Arcade target and with subset we have a replacement: `-subset libtests`.

Reply:
two more notes:
- The buildtests action never worked for coreclr and installer as installer always went with the subset paradigm and coreclr isn't msbuild driven yet for tests.
- The `/p:IncludeLibrariesTestSubset=true` switch is only temporary until we have better support for declaring to add to existing subsets, ie something like `-addsubset` but without a new switch, so hopefully something like `-subset "+libtests"`

Reply:
Ok, knowing this is a workaround makes me feel better 😄